lib/ansible/cli/config.py

```diff
@@ -370,18 +370,22 @@ def _render_settings(self, config):
 
         text = []
         for setting in sorted(config):
+            changed = False
             if isinstance(config[setting], Setting):
+                # proceed normally
                 if config[setting].origin == 'default':
                     color = 'green'
                 elif config[setting].origin == 'REQUIRED':
                     color = 'red'
                 else:
                     color = 'yellow'
+                    changed = True
                 msg = "%s(%s) = %s" % (setting, config[setting].origin, config[setting].value)
             else:
                 color = 'green'
                 msg = "%s(%s) = %s" % (setting, 'default', config[setting].get('default'))
-            if not context.CLIARGS['only_changed'] or color == 'yellow':
+
+            if not context.CLIARGS['only_changed'] or changed:  # and setting not in ('_terms', '_input')):
```

Comment:
Why is the last part commented out?

Reply:
cause it was special case, but now taken care of via 'required' instead